ds3-autogen-api/src/main/java/com/spectralogic/ds3autogen/api/models/docspec/Ds3DocSpec.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * ******************************************************************************
+ *   Copyright 2014-2016 Spectra Logic Corporation. All Rights Reserved.
+ *   Licensed under the Apache License, Version 2.0 (the "License"). You may not use
+ *   this file except in compliance with the License. A copy of the License is located at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   or in the "license" file accompanying this file.
+ *   This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ *   CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ *   specific language governing permissions and limitations under the License.
+ * ****************************************************************************
+ */
+
+package com.spectralogic.ds3autogen.api.models.docspec;
+
+import java.util.Optional;
+
+/**
+ * Contains the documentation for all commands and parameters
+ */
+public interface Ds3DocSpec {
+
+    /**
+     * Retrieves the documentation for the specified request
+     */
+    Optional<String> getCommandDocumentation(final String requestName);
```

Comment:
getCommandDocumentation doesn't jive with requestName param (change to getRequestDocumentation or change param to commandName)?